### PR TITLE
Added ping test for /logout controller #21

### PR DESCRIPTION
**Pull request by jstyle5**
### What this PR does (required):
- Issue Link: https://github.com/hatchways/team-breadsticks/issues/21
- Added ping test for /logout controller to check if it's connected to the server
- Configured ```package.json``` to be able to run tests
- Wrote how to run tests in the README.md

### Screenshots / Videos (required):
Screen Shot 2021-09-15 at 18 09 49

### Any information needed to test this feature (required):
- Go into the server directory ```cd server```
- Run ```npm test``` to run tests

### Any issues with the current functionality (optional):
- N/A

### CODE REVIEW

File: server/test/ping.spec.js [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
-        done();
-      });
+describe("Ping test with logout controller", () => {
+  it("should return 200 and message", async () => {
+    const testingMessage = { message: "Ping test with logout controller" };
+    const res = await chai.request.agent(app).post("/auth/logout").send(testingMessage);
+    res.should.have.status(200);
+    res.should.have
+      .property("text")
+      .eql(`Server is running. Message received: ${testingMessage.message}`);
```
Comment (ezeilo-su) [inline server/test/ping.spec.js:16]: :+1:

File: server/controllers/auth.js [hunk 2]
```diff
@@ -114,3 +114,10 @@ exports.logoutUser = asyncHandler(async (req, res, next) => {
 
   res.send("You have successfully logged out");
 });
+
+// @route POST /auth/logout
+// @desc Ping test with logout controller
+// @access Public
+exports.pingLogout = asyncHandler(async (req, res, next) => {
+  res.send(`Server is running. Message received: ${req.body.message}`);
+});
```
Comment (ezeilo-su) [inline server/controllers/auth.js:123]: :+1:

**Review by ezeilo-su — COMMENTED** [review]
Good work on adding this endpoint and writing spec for it. :+1:

File: README.md [hunk 3]
```diff
@@ -31,6 +31,12 @@ This is a little description about your project.
 
 ---
 
+## Test
+
+1. Go into the server directory `cd server`
+2. Run `npm test` to run tests
```
[the remark this answers is not in the document]
Comment (jstyle5) [reply README.md:37]: Applied

File: .gitignore [hunk 4]
```diff
@@ -14,7 +14,7 @@ node_modules/
 .env.development.local
 .env.test.local
 .env.production.local
-
+sample.env
```
Comment (rajivtitus) [inline .gitignore:17]: Please remove this from `gitignore` :)

**Review by rajivtitus — APPROVED** [review]
Great work! 👍🏽

File: .gitignore (hunk 4 above)
Comment (jstyle5) [reply .gitignore]: Applied

**Pull request merged**